apps/script/checkEntryPoints.js

```diff
@@ -57,6 +57,7 @@ const SILENCED = [
   'eval',
   'flappy',
   'gamelab',
+  'gamelab-api',
```

Comment:
Not PR feedback, but our discussion last week and now reading through this gets me thinking: The actual "Game Lab API" is actually duplicated in the gamelab and gamelab-api bundles.  Maybe what we eventually want is for the gamelab-api bundle to be reusable code, that could bind to a 'target platform' layer, gamelab-interpreted, gamelab-node or gamelab-native.

Reply:
yes, it is possible that we could do it that way. also of interest in the fact that `gamelab-api` (in this incarnation) is a subset of the existing `applab-api`. `applab-api` could be potentially broken apart, but the nature of how we use webpack suggests that we should probably bundle into one `js` file as much as possible for each set of APIs (otherwise, we'd end up with lots of duplicate JS code dependencies). In this sense, webpack behaves a little like a native code linker that pulls in exactly what you need and only one copy of each dependency. Once you include multiple webpack bundles, those benefits go away.